packages/lerna-semantic-release-pre/index.js

```diff
@@ -89,26 +99,87 @@ function bumpVersionCommitAndTag (nextRelease, done) {
   });
 }
 
+function updateDependencies (pkg, updates, dependenciesKey) {
+  var deps = pkg[dependenciesKey];
+
+  if (!deps) {
+    return pkg;
+  }
+
+  log.info('Updating ' + dependenciesKey + ' of ' + pkg.name);
+
+  pkg[dependenciesKey] = Object.keys(deps).reduce(function (dependenciesObj, depName) {
+    var update = updates[depName];
+
+    if (!update) {
+      // keep dependency version the same
+      dependenciesObj[depName] = deps[depName];
+      return dependenciesObj;
+    }
+
+    var oldVersion = deps[depName];
+    var newVersion = '^' + update.nextRelease.version;
```

Comment:
I wonder about this since it means we can't use other types of versioning – I think this makes the most sense, but is there some sort of semver library that'd let us make it support `>=` and fixed versions as well?